src/cli.rs

```diff
@@ -106,6 +115,12 @@ impl Options {
             3 | _ => options.log_level = log::LogLevelFilter::Trace
         }
 
+        if let Some(mut args) = matches.values_of("command") {
+            let command = String::from(args.next().unwrap());
```

Comment:
Looks like we can make this safer by using [`Arg::min_values()`](https://docs.rs/clap/2.20.0/clap/struct.Arg.html#method.min_values)